packages/Adapters/ClassicDTK/test/geometry/tstGeometryManager.cpp

```diff
@@ -76,69 +76,29 @@ TEUCHOS_UNIT_TEST( GeometryManager, geometry_manager_cylinder_test )
-
-    Teuchos::Array<BoundingBox> cylinder_boxes = 
-	geometry_manager.boundingBoxes();
-    Teuchos::ArrayRCP<unsigned long int> cylinder_gids = geometry_manager.gids();
-    for ( int i = 0; i < num_cylinders; ++i )
-    {
-	TEST_ASSERT( cylinder_gids[i] == gids[i] );
-
-	for ( int d = 0; d < 6; ++d )
-	{
-	    TEST_ASSERT( cylinder_boxes[i].getBounds()[d] ==
-			 cylinders[i].boundingBox().getBounds()[d] );
-	}
-
-	if ( cylinders[i].boundingBox().getBounds()[0] < x_min )
-	    x_min = cylinders[i].boundingBox().getBounds()[0];
-
-	if ( cylinders[i].boundingBox().getBounds()[1] < y_min )
-	    y_min = cylinders[i].boundingBox().getBounds()[1];
-
-	if ( cylinders[i].boundingBox().getBounds()[2] < z_min )
-	    z_min = cylinders[i].boundingBox().getBounds()[2];
-
-	if ( cylinders[i].boundingBox().getBounds()[3] > x_max )
-	    x_max = cylinders[i].boundingBox().getBounds()[3];
-
-	if ( cylinders[i].boundingBox().getBounds()[4] > y_max )
-	    y_max = cylinders[i].boundingBox().getBounds()[4];
-
-	if ( cylinders[i].boundingBox().getBounds()[5] > z_max )
-	    z_max = cylinders[i].boundingBox().getBounds()[5];
-    }
-
     BoundingBox local_box = geometry_manager.localBoundingBox();
-    TEST_ASSERT( local_box.getBounds()[0] == x_min );
-    TEST_ASSERT( local_box.getBounds()[1] == y_min );
-    TEST_ASSERT( local_box.getBounds()[2] == z_min );
-    TEST_ASSERT( local_box.getBounds()[3] == x_max );
-    TEST_ASSERT( local_box.getBounds()[4] == y_max );
-    TEST_ASSERT( local_box.getBounds()[5] == z_max );
+    TEST_EQUALITY( local_box.getBounds()[0], my_rank-1.0 );
+    TEST_EQUALITY( local_box.getBounds()[1], my_rank-1.0 );
+    TEST_EQUALITY( local_box.getBounds()[2], my_rank-0.5 );
+    TEST_EQUALITY( local_box.getBounds()[3], my_rank+100.0 );
+    TEST_EQUALITY( local_box.getBounds()[4], my_rank+100.0 );
+    TEST_EQUALITY( local_box.getBounds()[5], my_rank+99.5 );
 
     BoundingBox global_box = geometry_manager.globalBoundingBox();
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[0] , x_min-my_rank, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[1] , y_min-my_rank, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[2] , z_min-my_rank, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[3] , x_max+my_size-my_rank-1, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[4] , y_max+my_size-my_rank-1, 1.0e-14 );
-    TEST_FLOATING_EQUALITY( global_box.getBounds()[5] , z_max+my_size-my_rank-1, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[0] , -1.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[1] , -1.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[2] , -0.5, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[3] , my_size+99.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[4] , my_size+99.0, 1.0e-14 );
+    TEST_FLOATING_EQUALITY( global_box.getBounds()[5] , my_size+98.5, 1.0e-14 );
```

Comment:
why does ``local_box`` is tested ``TEST_EQUALITY`` while ``global_box`` uses ``TEST_FLOATING_EQUALITY``?